tests/integration/cmd_namespace_test.go

```diff
@@ -0,0 +1,28 @@
+package integration
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("create/delete/list/get/set namespace tests", func() {
+	var commonVar helper.CommonVar
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+	})
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+	for _, commandName := range []string{"namespace", "project"} {
+		When(fmt.Sprintf("using the alias %[1]s to list a %[1]s", commandName), func() {
+			It(fmt.Sprintf("should successfully list all the %s", commandName), func() {
+				helper.Cmd("odo", "list", commandName).ShouldPass()
```

Comment:
Maybe I'm being cautious again, but shouldn't we check the output here too, to make sure users have something displayed (at least the header) ? :-)